deploy/apostille.yaml

```diff
@@ -0,0 +1,167 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: apostille
+objects:
+# Service
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: ${{ NAME }}
+    labels:
+      app: ${{ NAME }}
+    annotations:
+      prometheus.io/scrape: 'true'
+  spec:
+    ports:
+      - name: default
+        protocol: TCP
+        port: ${{ CLUSTERIP_SERVICE_PORT  }}
+        targetPort: ${{ CLUSTERIP_SERVICE_TARGET_PORT }}
+    selector:
+      app: ${{ NAME }}
+    type: ClusterIP
+# Deployment
+- kind: Deployment
+  apiVersion: extensions/v1beta1
+  metadata:
+    name: ${{ NAME }}
+    labels:
+      app: ${{ NAME }}
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: ${{ NAME }}
+    template:
+      metadata:
+        labels:
+          app: ${{ NAME }}
+      spec:
+        volumes:
+          - name: apostille-config
+            secret:
+              secretName: apostille-server-secret
+              items:
+                - key: config_prod_json
+                  path: config.json
+          - name: apostille-signer-config
+            secret:
+              name: apostille-signer-config
+              items:
+                - key: signer_config_json
+                  path: config.json
+              defaultMode: 420
+          - name: apostille-server-secret
+            secret:
+              secretName: apostille-server-secret
+              items:
+                - key: tls_ca
+                  path: ca.crt
+                - key: tls_client_cert
+                  path: client.crt
+                - key: tls_client_key
+                  path: client.key
+                - key:
+          - name: apostille-signer-secret
+            secret:
+              secretName: apostille-signer-secret
+              items:
+                - key: client_ca_file
+                  path: ca.crt
+                - key: tls_cert_file
+                  path: cert.crt
+                - key: tls_key_file
+                  path: cert.key
+        containers:
+          - resources: {}
```

Comment:
Define `limits` and `requests` for each container. We should be able to configure them thru parameters.